mscore/menus.cpp

```diff
@@ -2248,9 +2248,8 @@ PalettePanel* MuseScore::newBreathPalettePanel()
 PalettePanel* MuseScore::newArpeggioPalettePanel()
       {
       PalettePanel* sp = new PalettePanel(PalettePanel::Type::Arpeggio);
-      sp->setName(QT_TRANSLATE_NOOP("Palette", "Arpeggios && Glissandi"));
-      sp->setGrid(27, 50);
-      sp->setDrawGrid(true);
+      sp->setName(QT_TRANSLATE_NOOP("Palette", "Arpeggios & Glissandi"));
+      sp->setGrid(27, 50);      sp->setDrawGrid(true);
```

Comment:
Brown Bag Warning: you removed line ending in the improper place.

Reply:
Outch, fixed